templates/components/matrix.build.hbs

```diff
@@ -0,0 +1,36 @@
+<div v-if="description" class="help-block description">\{{ description }}</div>
+<div class="fl-matrix">
```

Comment:
Add a `ref="matrix"` here so you can use `this.refs.matrix` to reference this node.

Reply:
done